### PR TITLE
Change Ktoml to TOML

### PR DESCRIPTION
**Pull request by Peanuuutz**
Didn't expect existing support for TOML but that one still doesn't implement some important features after months and is in slow progress. :(

### CODE REVIEW

**Review by qwwdfsad — COMMENTED** [review]
Hi, it seems like the library just hit its very first release -- congrats!

I would like to slightly postpone the merge to see that the format will be developed/released/used first.

**Comment by qwwdfsad** [comment]
>Didn't expect existing support for TOML but that one still doesn't implement some important features after months and is in slow progress. :(

By the way, It may be the case that it will be much easier and more impactful to cooperate with the author of `ktoml` and open a few PRs. Maintaining a library is time-consuming work and maybe here it will be better to collaborate to cut the costs instead of having two identical libraries

**Comment by Peanuuutz** [comment]
> By the way, It may be the case that it will be much easier and more...

Thanks for review! I'll try to build up collaboration. :blush:

**Comment by Peanuuutz** [comment]
Changed my mind. Just fix the issue that `Ktoml` is not the actual format, `TOML` is.

File: formats/README.md [hunk 1]
```diff
@@ -59,13 +59,12 @@ This library allows serialization and deserialization of objects to and from [Av
...
 
-### Ktoml 
+### TOML (Multiplatform) 
 * GitHub repo: [akuleshov7/ktoml](https://github.com/akuleshov7/ktoml)
 * Artifact ID: `com.akuleshov7:ktoml-core`
-* Platforms: multiplatform, all Kotlin supported platforms
+* Platforms: all supported platforms
 
-Fully Native and Multiplatform Kotlin serialization library for serialization/deserialization of TOML format.
-This library contains no Java code and no Java dependencies and it implements multiplatform parser, decoder and encoder of TOML.
```
Comment (sandwwraith) [inline formats/README.md:68]: Isn't this line also relevant?

Comment (Peanuuutz) [reply formats/README.md:68]: Yes, just want to make the title be like others.

Comment (sandwwraith) [reply formats/README.md:68]: Not all format descriptions fit in one line or sentence (see Avro or XML), so IMO it's better to leave it as is

**Pull request closed without merging**